plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html

```diff
@@ -75,6 +75,18 @@ <h2 id="Threat">Threat</h2>
 				<td>Tampering attacks refer to many forms of sabotage (intentional modification of the information system in a way that makes them harmful).</td>
 			</tr>
 		</table>
+		<p>A 
+			<i>Threat</i> kind could be full customize by the user in Cybersecurity Configuration.
```

Comment:
The threat kind can be customized by the user in the Cybersecurity Configuration, by editing the Threat.Kind type.